### PR TITLE
Jetpack App: Update filter bar selected color for Jetpack

### PR DESCRIPTION
**Pull request by momo-ozawa**
## Description

This PR updates the filter bar selected color to use `.primary` for both light/dark mode

Design ref: pcdRpT-k6-p2
> On iOS: Change segmented controls (top tabs?) to use black text with a green line (rather than green text with a green line).

Before | After
-- | --
 | 

## How to test
1. Go to My Site
2. Tap on Stats
3. Turn on dark mode
4. 👀 Notice the filter bar selected color is green

## Regression Notes
1. Potential unintended areas of impact
Filter bar selected color in light mode for the Jetpack App

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested and confirmed that the filter bar selected color looks correct

3. What automated tests I added (or what prevented me from doing so)
None

## PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by momo-ozawa** [comment]
Hi @osullivanchris 👋 
Which colors should we be using for the Views, Visitors, Likes, Comments tabs on the Stats screen for light/dark mode? Currently both the text and the selected tab use `.primary` (i.e. Jetpack Green 50).

**Review by emilylaguna — APPROVED** [review]
LGTM! 🚀 once Chris approves.

**Comment by osullivanchris** [comment]
> Which colors should we be using for the Views, Visitors, Likes, Comments tabs on the Stats screen for light/dark mode? Currently both the text and the selected tab use .primary (i.e. Jetpack Green 50).

👋  The 'after' screenshot looks correct. Intention was 
- light mode: black text with primary green underline
- dark mode: white text with primary green underline 

Looks like that's what you have, thanks 😄

**Pull request merged**
